ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -2856,13 +2861,15 @@ struct TImportInfo: public TSimpleRefCount<TImportInfo> {
 
         EState State = EState::GetScheme;
         ESubState SubState = ESubState::AllocateTxId;
+        EChangefeedState ChangefeedState = EChangefeedState::CreateChangefeed;
         TTxId WaitTxId = InvalidTxId;
         TActorId SchemeGetter;
         TActorId SchemeQueryExecutor;
         int NextIndexIdx = 0;
         int NextChangefeedIdx = 0;
         TString Issue;
         int ViewCreationRetries = 0;
+        NKikimr::TPathId StreamImplPath;
```

Comment:
По аналогии с `DstPathId`:
```suggestion
        NKikimr::TPathId StreamImplPathId;
```